src/Components/Alert/Components/Filters/__tests__/ArtistSeries.jest.tsx

```diff
@@ -0,0 +1,50 @@
+import { screen } from "@testing-library/react"
+import { setupTestWrapperTL } from "DevTools/setupTestWrapper"
+import { ArtistSeriesQueryRenderer } from "Components/Alert/Components/Filters/ArtistSeries"
+import { flushPromiseQueue } from "DevTools/flushPromiseQueue"
+import { ArtistSeriesOptionsQuery$data } from "__generated__/ArtistSeriesOptionsQuery.graphql"
+
+jest.unmock("react-relay")
+
+describe("ArtistSeries", () => {
+  it("renders artist series options", async () => {
+    const { renderWithRelay } = setupTestWrapperTL({
+      Component: () => {
+        return <ArtistSeriesQueryRenderer />
```

Comment:
since we are using the alerts context in the artist series component, we would need to wrap the component with `<AlertProvider>`